grizli/model.py

```diff
@@ -1606,9 +1612,10 @@ def __init__(self, sci=None, err=None, dq=None,
 
         self.wcs = None
 
-        if (instrument in ['NIRISS', 'NIRCAM']) & (~self.is_slice):
```

Comment:
why is this removed?

Reply:
This was something I commented out while debugging and forgot to change back. Fixed here: 46eab444d0e74967c8198972f4ea6be2a638cf2f